upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -668,7 +1129,7 @@ data:
       },
       "outputPaths": ["stdout"],
       "errorOutputPaths": ["stderr"],
-      "encoding": "json",
+      "encoding": "console",
```

Comment:
Why not use `json` encoding which is more standard for logs management?

Reply:
Makes logs very annoying to read and everything else is plain text. No preference here though. @olemarkus any thoughts?

Reply:
We can add a `karpenter.logFormat` variable with `console` by default. 
I can add it after you merge this PR if you agree. :slightly_smiling_face:

Reply:
Sure, that would be nice. Thanks!